app/views/header.js

```diff
@@ -80,6 +86,13 @@ const Header = Mn.ItemView.extend({
         application.channel.trigger('upnext:reset');
     },
 
+    toggleDrawer(e) {
+        e.preventDefault();
+        let drawer = document.querySelector('[role="toolbar"]')
```

Comment:
Same here: you should only trigger an event `drawer:toggle` and react to it in the dedicated view.